### PR TITLE
UI/login

### PR DESCRIPTION
**Pull request by YasuyoshiWada**

### CODE REVIEW

File: public/mix-manifest.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
     "/js/app.js": "/js/app.js",
     "/css/app.css": "/css/app.css",
+<<<<<<< HEAD
+    "/css/login-forget.css": "/css/login-forget.css",
```
Comment (Ninit29) [inline public/mix-manifest.json:5]: Please resolve conflict

File: webpack.mix.js [hunk 2]
```diff
@@ -13,5 +13,10 @@ const mix = require('laravel-mix');
 
 mix.js('resources/js/app.js', 'public/js')
     .sass('resources/sass/app.scss', 'public/css')
+<<<<<<< HEAD
+    .sass('resources/sass/login.scss', 'public/css')
```
Comment (Ninit29) [inline webpack.mix.js:17]: Please resolve conflict

File: public/mix-manifest.json (hunk 1 above)
Comment (YasuyoshiWada) [reply public/mix-manifest.json:5]: FD9E0B6E-5E33-4FAC-B698-9C33DDE3D29E
Do you mean to remove these when I push?" /css/login-forget.css":"/css/login-forget.css",

Comment (Ninit29) [reply public/mix-manifest.json:5]: i mean this file mix-manifest.json

**Comment by YasuyoshiWada** [comment]
I deleted public/mix-manifest.json
& webpack.mix.js.
Please check.

**Pull request merged**